pylzy/lzy/api/v1/workflow.py

```diff
@@ -167,6 +171,8 @@ def __exit__(self, exc_type, exc_val, exc_tb) -> None:
         finally:
             if exc_type is None:
                 self.__destroy()
+            elif exc_type is AioRpcError:
```

Comment:
Here may be UNAVAILABLE, so we must abort wf in this case

Reply:
UNAVAILABLE is retryable error. Python lzy-client should repeat request in case. If after an amount of attempts lzy service is still unavailable so I believe that it is really UNAVAILABLE and an abort request is useless :)